brainfuck/brainfuck.d

```diff
@@ -45,10 +47,11 @@ class Program {
       pc++;
       code ~= c;
     }
+    bracket_map.rehash;
   }
 
   void run() {
-    Tape tape = new Tape();
+    auto tape = Tape(new int[100]);
```

Comment:
I know you had concerns around this line, but the amount of tape used by these tests is around 8 bytes.   Pre-allocating a small region like this doesn't change the benchmarks.  It's only to make sure that the tape is initialized properly since structs do not allow default constructors to be specified in DLang.